grid-client/integration_tests/vms_private_ips_test.go

```diff
@@ -42,15 +42,10 @@ func TestDeploymentsDeploy(t *testing.T) {
 	network, err := generateBasicNetwork([]uint32{nodeID})
 	require.NoError(t, err)
 
-	vm1 := workloads.VM{
-		Name:        vm1Name,
-		NodeID:      nodeID,
-		NetworkName: network.Name,
-		CPU:         minCPU,
-		MemoryMB:    minMemory * 1024,
-		Flist:       "https://hub.grid.tf/tf-official-apps/threefoldtech-ubuntu-22.04.flist",
-		Entrypoint:  "/sbin/zinit init",
-	}
+	vm1, err := generateBasicVM(vm1Name, nodeID, network.Name, "")
+	require.NoError(t, err)
+
+	vm1.Flist = "https://hub.grid.tf/tf-official-apps/threefoldtech-ubuntu-22.04.flist"
```

Comment:
you can define ubuntu flist as a global variable and reuse it as you like, I see you used it multiple times